proofs/alf/cvc5/programs/Strings.smt3

```diff
@@ -599,18 +786,31 @@
   )
 )
 
-; `$str_mk_re_loop_elim n d r` returns the reduction of the regular expression
-;   (re.loop n n+d r)
-; where d>0. This is the regular expression:
-;   (re.union r^n ... r^{n+d})
-; where r^k is (re.++ r ... r), where r is repeated k times when k>1, or
-; r if k=1 or @re.empty if k=0.
+; define: $str_mk_re_loop_elim
+; args:
+; - n Int: The minimum number of iterations for a re.loop term.
+; - d Int: The difference between the min and max iterations for a re.loop term.
+; - r RegLan: The regular expression argument of re.loop.
+; return: >
+;   The reduction of the regular expression
+;     (re.loop n n+d r)
+;   where d>0. This is the regular expression:
+;     (re.union r^n ... r^{n+d})
+;   where r^k is (re.++ r ... r), where r is repeated k times when k>1, or
+;   r if k=1 or @re.empty if k=0.
 (define $str_mk_re_loop_elim ((n Int) (d Int) (r RegLan))
   ($singleton_elim ($str_mk_re_loop_elim_rec n d r @re.empty))
 )
 
-; Helper for ProofRewriteRule::STR_IN_RE_CONCAT_STAR_CHAR.
-; Note that we do not require a singleton elimination step since the original input should be a concatenation term.
+; program: $str_mk_str_in_re_concat_star_char
+; args:
+; - s String: The string term to process.
+; - r RegLan: The regular expression to process.
+; return: >
+;   The conclusion for ProofRewriteRule::STR_IN_RE_CONCAT_STAR_CHAR for
+;   (str.in_re s r), where r is assumed to be a re.* whose body has fixed length
+;   of one. Note that we do not require a singleton elimination step for and
```

Comment:
```suggestion
;   of one. Note that we do not require a singleton elimination step for s
```

Reply:
"and" is correct here, it is referred to the function `and`.